### ISSUE TITLE
no-array-callback-reference

### ISSUE THREAD

**Issue by GuoSirius** [sindresorhus/eslint-plugin-unicorn#1006]
![image](https://user-images.githubusercontent.com/48358523/104254008-d5084b80-54b0-11eb-8ef0-03c5d407bf0d.png)

#701

**Comment by GuoSirius** [comment]
Element UI Tree Component method

**Comment by sindresorhus** [comment]
You yet again have not provided enough information. Please try harder.

**Comment by GuoSirius** [comment]
```javascript
this.$refs.tree.filter(val)
```

像这种 调用，可能是一个组件或者对象方法里自定义的filter方法，但是这个方法仅仅传递一个值而不是一个函数，但是在校验规则里面就会报错，这是不被期望的。

是否可以修正，或者加一个配置项，比如 
```javascript
'unicorn/no-array-callback-reference': { ignorePrefix: 'this.$refs.tree' }
```

**Comment by GuoSirius** [comment]
```javascript
this.$refs.tree.filter(val)
```

A call like this might be a custom filter method in a component or object method that passes only a value rather than a function, but will return an error in the validation rule, which is not expected.

Can you fix this or add a configuration item such as 
```javasctipt
'unicorn/no-array-callback-reference': { ignorePrefix: 'this.$refs.tree' }
```

**Comment by Vages** [comment]
This is the expected behavior from the rule. There are discussions about extending the rule and adding more behavior, but this issue is in and of itself not a bug.
